tests/Command/MinifyAssetCommandTest.php

```diff
@@ -65,6 +65,20 @@ public function testMinifyAssetCommandFailsWhenInputFileDoesNotExist(): void
         $this->assertStringContainsString('Cannot read file', $tester->getDisplay());
     }
 
+    public function testMinifyAssetCommandFailsWhenInputFileIsNotCssOrJS(): void
+    {
+        $minifier = $this->createMock(MinifierInterface::class);
+
+        $command = new MinifyAssetCommand($minifier, __DIR__.'/../Fixtures');
+        $tester = new CommandTester($command);
+
+        $tester->execute(['input' => 'MinifyBundleTestKernel.php']);
+        $this->assertSame(Command::FAILURE, $tester->getStatusCode());
+        $display = $tester->getDisplay();
+        $this->assertStringContainsString('The type of', $display);
+        $this->assertStringContainsString('TestKernel.php" is "php", it must be "css" or "js".', $display);
```

Comment:
The error message shows the full path, it depends on the environment where tests are run, I skipped it. Using a regex was too cumbersome because there automatic line breaks. Providing the max width of the terminal would add a dependency on the behaviour of Symfony's console component that is unnecessary here.